core/src/test/java/org/elasticsearch/discovery/zen/ZenDiscoveryIT.java

```diff
@@ -278,7 +278,7 @@ public void testHandleNodeJoin_incompatibleMinVersion() throws UnknownHostExcept
         Settings nodeSettings = Settings.builder()
                 .put("discovery.type", "zen") // <-- To override the local setting if set externally
                 .build();
-        String nodeName = internalCluster().startNode(nodeSettings, Version.V_5_0_0_alpha1);
+        String nodeName = internalCluster().startNode(nodeSettings, Version.V_5_0_0);
```

Comment:
I'm good with latest. It's also probably a good idea to change testJoinElectedMaster_incompatibleMinVersion as well